lib/content-services/search/components/search-filter/search-filter.component.ts

```diff
@@ -342,4 +356,55 @@ export class SearchFilterComponent implements OnInit, OnDestroy {
 
         return `${fieldName}:${startLimit}"${start}" TO "${end}"${endLimit}`;
     }
+
+    private getApiResponseField(context: ResultSetContext, itemType: string, fieldLabel: string): GenericFacetResponse {
+        return (context.facets || []).find((response) => response.type === itemType && response.label === fieldLabel) || {};
+    }
+
+    private getAlreadyDisplayedField(itemType: string, fieldLabel: string): FacetField {
```

Comment:
Rename to better show what it does
```suggestion
    private findResponseFacet(itemType: string, fieldLabel: string): FacetField {
```

Reply:
I applied the suggested renamings. Thank you, @DenysVuika